doc/introduction/photonic_hardware.rst

```diff
@@ -6,9 +6,9 @@ photonic hardware or to be run on a cloud simulator via the Xanadu Cloud.
 
 .. warning::
 
-    An API key is required to access the Xanadu Cloud platform. If you do not
-    have a Xanadu Cloud API key, you can request hardware access `via the sign
-    up form on the Xanadu website <https://xanadu.ai/access>`__.
+    An API key is required to communicate with the hardware locally.
+    Set up a `Xanadu Cloud <cloud.xanadu.ai>` account for free, and generate 
```

Comment:
You might need to add a little https in front of the link. I noticed that it the XC links didn't work locally for me when testing yesterday.
```suggestion
    Set up a `Xanadu Cloud <https://cloud.xanadu.ai>` account for free, and generate 
```

Reply:
<img width="772" alt="Screen Shot 2022-06-02 at 10 38 57 AM" src="https://user-images.githubusercontent.com/6934626/171654874-5ab816c9-e806-468c-b979-922b40011f5f.png">